### ISSUE TITLE
Shigure Tekko +1 Rainy Weather Effect Does Not Activate

### ISSUE THREAD

**Issue by Daruvan** [LegionDark/Issues#1499]
**Date & Time**: 5/29/2017

**FFXI Client Version (use `/ver`)**: 30170304_1

**Server's Expected Client Version matches, yes/no? (use `$ecv`)**: Yes

**Character Name**:  Daru

**Nation**:  Windurst

**Job(level)/Sub Job(level)**: SAM99 / THF49

**NPC or Monster or item Name**: Shigure Tekko +1

**Zone name**: Any

**Coordinates (use `$where`)**: Anywhere

**ffxiah.com link (for items issues only)**: http://www.ffxiah.com/item/27996/shigure-tekko-1

**Multi-boxing? (multiple clients on same connection)**: Yes

**Steps To Reproduce / any other info**: Rainy weather / Rainstorm do not activate Attack+30, Accuracy+30 effect.
![1](https://cloud.githubusercontent.com/assets/21298141/26566914/eafff0b4-44c4-11e7-93f9-c39f40c80802.PNG)

**Comment by R3P0FFXI** [comment]
@Daruvan No mods exist for this in DSP sorry

**Comment by Daruvan** [comment]
Ok, thank you!

**Comment by TeoTwawki** [comment]
@R3P0FFXI YES THEY DO. Its a LATENT not a freaking mod. I'm tired of saying this.

https://github.com/DarkstarProject/darkstar/blob/master/sql/item_latents.sql#L16

https://github.com/DarkstarProject/darkstar/blob/master/src/map/latent_effect.h#L80
